backend/build.gradle

```diff
@@ -6,6 +6,22 @@ sourceCompatibility = 1.8
 sourceSets.main.java.srcDirs = ["src/main"]
 sourceSets.test.java.srcDirs = ["src/test"]
 
+project.ext.mainClassName = "nl.tudelft.ti2806.riverrush.desktop.MainDesktop"
```

Comment:
Should be: " project.ext.mainClassName = "nl.tudelft.ti2806.riverrush.backend.MainBackend" "